### PR TITLE
build: fetch the main branch

### PR DESCRIPTION
**Pull request by bcoe**
It looks like the checkout library doesn't fetch the main branch by default (it's probably a shallow clone?).

### CODE REVIEW

File: .github/workflows/interrogate.js [hunk 1]
```diff
@@ -16,6 +16,11 @@ const {execSync} = require('child_process');
 execSync('git fetch origin master');
 const baseRef = process.env.GITHUB_BASE_REF;
 let status;
+
+execSync('git fetch origin master');
```
Comment (tbpg) [inline .github/workflows/interrogate.js:20]:
```suggestion
execSync('git fetch --depth 2 origin master');
```

**Review by tbpg — CHANGES_REQUESTED** [review]

File: .github/workflows/interrogate.js (hunk 1 above)
Comment (tmatsuo) [inline .github/workflows/interrogate.js:20]: It will break once github changes the default branch from `master` to something else.

Comment (bcoe) [reply .github/workflows/interrogate.js]: we re no longer using a backed in vlue.

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -14,6 +14,8 @@ jobs:
       requiredJobs: ${{ steps.interrogate.outputs.requiredJobs }}
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```
Comment (tbpg) [inline .github/workflows/ci.yaml:18]: Why not 2 or some small number? Seems like this does the entire repo?

Comment (bcoe) [reply .github/workflows/ci.yaml:18]: @tbpg I tried a variety of other values and was consistently getting an error that HEAD~ was ambiguous. The docs indicate that setting `0` "Fetch all history for all tags and branches".

**Review by tmatsuo — APPROVED** [review]

**Review by JustinBeckwith — APPROVED** [review]

**Review by tbpg — APPROVED** [review]
Thank you for fixing this!

**Pull request merged**